data/human/free worlds epilogue.txt

```diff
@@ -197,3 +197,96 @@ mission "FW Epilogue: Danforth"
 			label end
 			`	After you talk for a while longer, Danforth wishes you luck and sends you on your way. "I'll be in touch if we ever need your assistance in the future," he says.`
 				decline
+
+
+
+mission "FW Epilogue: Barmy"
+	minor
+	landing
+	name `Weapon Testing`
+	description `A test ship with new weaponry is in orbit around <planet>. Fight and disable it (but do not board or destroy it!) and then land to give Barmy Edward feedback on its performance.`
+	source "Rust"
+	to offer
+		has "free worlds plot completed"
+	on offer
+		conversation
+			`As you complete your landing procedure, you recall having come here to assist in some testing for new weapons, back in the thick of the war. Do you want to check in on Barmy Edwards, the lead engineer?`
+			choice
+				`	(Yes.)`
+				`	(No.)`
+					defer
+			scene "scene/loc0"
+			`	You give your name to the receptionist, and she directs you to a hangar where you find Barmy Edward, who is working on a weapon that looks like a modified plasma cannon. When he notices you, he gets up from his work and shakes your hand warmly. "Ah, <last>. It's been too long! How fortunate for me! I'm actually just putting the finishing touches on a new prototype. You wouldn't be too busy to assist me in some practical testing, would you?`
+			choice
+				`	"I'm not sure about this. What would you have me do?"`
+				`	"Being a target for experimental weapons? Just tell me what you need!"`
+			`	"Not much out of the ordinary from our tests in the past," he replies. "You'll find a ship of ours in orbit that goes by 'Hunk'. It's been fitted with a few of these new weapons. Your job will be to fight it and disable it, then tell us what you think of its combat capabilities, and transfer your ship's logs so we can analyze the fight."`
```

Comment:
```suggestion
			`	"Not much out of the ordinary from our tests in the past," he replies. "You'll find a ship of ours in orbit that goes by 'Hunk'. It's been fitted with a few of these new weapons. Your job will be to fight and disable it, tell us what you think of its combat capabilities, and transfer your ship's logs so we can analyze the fight."`
```
This last sentence is kind of tough to work out grammatically, but I think this takes out all the bumps.